### DISCUSSION TITLE
Is there any reason why Asset.is_animated isn't a property?

### DISCUSSION THREAD

**Discussion by itswilliboy** [Rapptz/discord.py#9539] [General] [answered]
Is there any reason why [`discord.Asset.is_animated`](https://discordpy.readthedocs.io/en/stable/api.html?highlight=is_animated#discord.Asset.is_animated) is not a `@property`?

https://github.com/Rapptz/discord.py/blob/master/discord/asset.py#L355-L357

**Comment by Rapptz** [comment, marked as answer]
None of the `is_` methods are properties. This is consistent design within the lib, so once you see `is_` you know it's a method to be called.

**Reply by itswilliboy** [reply]
Ah, gotcha. Thanks for the explanation!
